incubator/virtualcluster/config/setup/all_in_one.yaml

```diff
@@ -233,6 +233,8 @@ spec:
       containers:
       - command:
         - manager
+        args:
+        - --disable-stacktrace=true
```

Comment:
Should this also be in - https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/virtualcluster/config/setup/all_in_one_aliyun.yaml

Reply:
Since this is specific to aliyun, should it be in the repo?

Reply:
With stack trace, the error message is quite verbose, and I found it isn't helpful for debugging, somthing like following: 
```bash
{"level":"error","ts":1597882807.380376,"logger":"virtualcluster-controller","caller":"zapr/zapr.go:128","msg":"fail to create virtualcluster","vc":"vc-sample-1","retrytimes":1,"error":"desired ClusterVersion cv-sample-np not found","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/Users/c.zheng/Work/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128\nsigs.k8s.io/multi-tenancy/incubator/virtualcluster/pkg/controller/virtualcluster.(*ReconcileVirtualCluster).Reconcile\n\t/Users/c.zheng/Work/go/src/github.com/kubernetes-sigs/multi-tenancy/incubator/virtualcluster/pkg/controller/virtualcluster/virtualcluster_controller.go:183\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/Users/c.zheng/Work/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.1/pkg/internal/controller/controller.go:233\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/Users/c.zheng/Work/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.1/pkg/internal/controller/controller.go:209\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker\n\t/Users/c.zheng/Work/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.1/pkg/internal/controller/controller.go:188\nk8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1\n\t/Users/c.zheng/Work/go/pkg/mod/k8s.io/apimachinery@v0.18.6/pkg/util/wait/wait.go:155\nk8s.io/apimachinery/pkg/util/wait.BackoffUntil\n\t/Users/c.zheng/Work/go/pkg/mod/k8s.io/apimachinery@v0.18.6/pkg/util/wait/wait.go:156\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/Users/c.zheng/Work/go/pkg/mod/k8s.io/apimachinery@v0.18.6/pkg/util/wait/wait.go:133\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/Users/c.zheng/Work/go/pkg/mod/k8s.io/apimachinery@v0.18.6/pkg/util/wait/wait.go:90"}
```
and the value of the caller field is always "zapr/zapr.go:128", which is useless. By disabling the stacktrace and the caller field, the error log will be more clear
```bash
{"level":"error","ts":1597882807.380376,"logger":"virtualcluster-controller","msg":"fail to create virtualcluster","vc":"vc-sample-1","retrytimes":1,"error":"desired ClusterVersion cv-sample-np not found"}
```

Reply:
Yeah, 100% agree this is much nicer. I've never found the stacktraces helpful.